index.js

```diff
@@ -22,10 +23,32 @@ types.forEach(function (type) {
   postNettyVersionsByProtocolVersion[type] = indexer.buildIndexFromArrayNonUnique(protocolVersions[type].filter(function (e) { return e.usesNetty }), 'version')
 })
 
+function Version (type, version, majorVersion) {
+  const versions = versionsByMinecraftVersion[type]
+  // Allows comparisons against majorVersion even if `other` is not present in the versions.json (e.g. 1.17.0 exists but not 1.17)
+  for (const version in versions) {
+    const ver = versions[version]
+    versions[ver.majorVersion] = versions[ver.majorVersion] || ver
+  }
+  // TODO: Data for Minecraft classic is missing in protocolVersions.json, move this to its own type ?
+  const v1 = versions[version]?.dataVersion ?? 0
```

Comment:
We already have isNewerOrEqualTo
Can you either use that either make them alias of each other so it's not breaking ?

Reply:
@extremeheat did you see this comment ? can you fix it ?

Reply:
Yeah I saw , do you think these should be removed or should isNewerOrEqual should alias to these ? The purpose of this was to encapsulate the version related data (type, version, dataVersion, etc) into one object instead of relying on root mcData.

Reply:
So actually looking at the actual code, I do not understand the design.
The point of isNewerOrEqual is you don't need to load the 2 versions to be able to compare them.

Reply:
It’s not loading the version though? It’s just comparing versionsByMinecraftVersion[type].dataVersion just like isNewerOrEquals to. The for loop doesn’t need to exist, it just makes aliases so for example 1.17 and 1.17.0 resolve to the same thing (on bedrock the version has a .0 at the end which fails the comparisons)

Reply:
ah I see, you did 2 separate changes, one with this Version and another putting it in each loaded data

Reply:
ok, so could you
1. add this to the doc
2. make isNewerOrEqual use this code